src/flb_output.c

```diff
@@ -1471,6 +1471,14 @@ int flb_output_log_check(struct flb_output_instance *ins, int l)
 int flb_output_upstream_set(struct flb_upstream *u, struct flb_output_instance *ins)
```

Comment:
Same as in `flb_input_upstream_set` with the additional question "why are we duplicating this code?"